src/pages/splash/splash.ts

```diff
@@ -24,15 +25,48 @@ export class SplashPage {
     public navParams: NavParams,
     public storage: StorageService,
     private controller: HomeController,
-
+    private globalization: Globalization,
     private kafka: KafkaService) {
     const parentPage = this.navParams.data.parentPage
     if(parentPage){
       console.log(`VIEW ${parentPage}`)
       this.hasParentPage = true
     }
     this.status = 'Updating notifications...'
-    this.controller.setNextXNotifications(DefaultNumberOfNotificationsToSchedule)
+    Promise.all([this.storage.get(StorageKeys.TIME_ZONE),
+      this.storage.get(StorageKeys.UTC_OFFSET)])
+    .then(([timeZone, utcOffset]) => {
+      this.globalization.getDatePattern( { formatLength: 'short', selector: 'date and time'} )
+      .then((res) => {
+        // cancel all notifications if timezone/utc_offset has changed
+        if(timeZone != res.timezone || utcOffset != res.utc_offset)
+        {
+          console.log('[SPLASH] Timezone has changed to '+ res.timezone + '. Cancelling notifications!')
+          this.storage.set(StorageKeys.TIME_ZONE, res.timezone)
+          this.storage.set(StorageKeys.UTC_OFFSET, res.utc_offset)
+          this.controller.cancelNotifications()
```

Comment:
depending on how the delay when Timezone changes, we need to re-schedule the questionnaires. We should trigger refresh config

Reply:
Done above the block